api/dps/index.go

```diff
@@ -31,12 +31,12 @@ import (
 // machines across a network.
 type Index struct {
 	client APIClient
-	codec  index.Codec
+	codec  dps.Codec
```

Comment:
I thought about making the same change on the access-api feature branch but I thought it wasn't worth a PR, nice to see that you did it here 👍